gk/main.c

```diff
@@ -46,9 +46,11 @@
 
 #include "bpf.h"
 
-#define	START_PRIORITY		 (38)
+#define	START_PRIORITY		(38)
 /* Set @START_ALLOWANCE as the double size of a large DNS reply. */
-#define	START_ALLOWANCE		 (8)
+#define	START_ALLOWANCE		(8)
+/* Number of seconds before this request entry expires. */
+#define REQ_EXPIRATION_SEC	(4)
```

Comment:
This line should align with the above two ``#define`` declarative. Moreover, a brief explanation on why it is ``4``.

Reply:
Thanks, Qiaobin. I was incorrectly using a space instead of a tab between the `#define` and the name, but also note there were extra spaces in front of the existing two values `(32)` and `(8)` that I removed, so the diff will show this.